mlflow/java/client/src/main/java/org/mlflow/tracking/MlflowClient.java

```diff
@@ -636,4 +638,81 @@ private ArtifactRepository getArtifactRepository(String runId) {
     URI baseArtifactUri = URI.create(getRun(runId).getInfo().getArtifactUri());
     return artifactRepositoryFactory.getArtifactRepository(baseArtifactUri, runId);
   }
+
+
+  // ********************
+  // * Model Repository *
+  // ********************
+
+  /**
+   * Return the latest model version for each stage (devo, staging, prod).
```

Comment:
Currently supported stages are : `None`, `Staging`, `Production`, and `Archived`

Reply:
I've updated the docs with those stages. Would it be better to link to a doc about staging instead?

Reply:
The agreement is remove the documentation about the specific stages and ask customers to call the getStages API